packages/x-charts/src/ScatterChart/Scatter.tsx

```diff
@@ -94,7 +95,7 @@ function Scatter(props: ScatterProps) {
           key={dataPoint.id}
           cx={0}
           cy={0}
-          r={markerSize}
+          r={(dataPoint.isHighlighted ? 1.2 : 1) * markerSize}
```

Comment:
```suggestion
          r={markerSize}
```

Reply:
Note: Highcharts also do increase the point radius, when highlighted, but that is probably a separate story/functionality and would need to be controllable. 🤷 🙈